examples/kubernetes/vitess-up.sh

```diff
@@ -103,6 +105,9 @@ echo "*  VTGate count: $vtgate_count"
 echo "*  Cells: $cells"
 echo "****************************"
 
+echo 'Generating config.sh'
+echo -e "\n\n$BACKUP_BUCKET" | ./configure.sh
```

Comment:
It would be nice to let vitess-up.sh still be used with an already-generated config.sh. Would you be willing to run configure.sh as a separate step before vitess-up.sh? We could have a batch mode for configure.sh that skips prompts and gets values from env vars. If any required env vars aren't specified when in batch mode, it should exit with a nonzero code. That would be less fragile when we start adding more to configure.sh.